recipes/clinker/meta.yaml

```diff
@@ -0,0 +1,44 @@
+
+{% set name = "clinker" %}
+{% set version = "1.32" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Oshlack/Clinker/archive/v{{ version }}.tar.gz
+  sha256: "5dbb97da27f29f53a8282e456ba0093bbdde292eb9e76405434117da4d1a76be"
+
+build:
+  number: 1
+  noarch: generic
+
+requirements:
+  run:
+    - bpipe
+    - samtools
+    - star ==2.5.3a
+    - bioconductor-biomart ==2.34.2
+    - bioconductor-gviz ==1.22.3
```

Comment:
If you need to pin that tightly, you should distribute an environment file. Please make these something like 

```
 - star >=2.5.3a, <3
 - bioconductor-biomart >=2.34.2, <3
 - bioconductor-gviz >=1.22.3, <2
```